QuickBooksSharp/Services/DataService.cs

```diff
@@ -112,12 +112,17 @@ public async Task<IntuitResponse<BatchItemResponse[]>> BatchAsync(IntuitBatchReq
             };
         }
 
-        /// <summary>
-        /// Create, Update, or SparseUpdate the entity, depending on the value of the 'sparse' property
-        /// </summary>
-        public async Task<IntuitResponse<TEntity>> PostAsync<TEntity>(TEntity e) where TEntity : IntuitEntity
+        /// <inheritdoc/>
+        public async Task<IntuitResponse<TEntity>> PostAsync<TEntity>(TEntity e, OperationEnum include = OperationEnum.Unspecified) where TEntity : IntuitEntity
         {
-            var res = await _client.PostAsync<IntuitResponse>(new Url(_serviceUrl).AppendPathSegment(GetEntityName(typeof(TEntity))), e);
+            var url = new Url(_serviceUrl).AppendPathSegment(GetEntityName(typeof(TEntity)));
+
+            if (include != OperationEnum.Unspecified)
+            {
+                url = url.SetQueryParam("include", include);
```

Comment:
@fbridger why is the param called 'include'. As far as I can see in the docs, it should be called 'operation'